plugins/module_utils/util.py

```diff
@@ -366,7 +366,7 @@ def normalize_healthcheck(healthcheck, normalize_test=False):
                 continue
             if key in duration_options:
                 value = convert_duration_to_nanosecond(value)
-            if not value:
+            if not value and key != 'test':
                 continue
```

Comment:
This is problematic, as it modifies behavior even if `test_cli_compatible` is not set. Please change this so it does **not** modify any behavior if `test_cli_compatible` is not explicitly `True`.

Same below.

Reply:
Absolutely, I've changed that.